### ISSUE TITLE
Nothing happened in the development environment. What am I doing wrong or missing?

### ISSUE THREAD

**Issue by bloggerklik** [oblador/react-native-store-review#68]
I have an app published on Google Play. I did the following in the development environment but nothing happened. I tested it on real device. My Google Play account is registered to the test account. Nothing happened in the development environment. What am I doing wrong or missing? Thanks.

- I downloaded.
npm i react-native-store-review

- I imported.
import * as StoreReview from 'react-native-store-review';

- Code
```
    React.useEffect(() => {
        StoreReview.requestReview();
    }, []);
```

**Comment by AlexCernik** [comment]
Hi @bloggerklik  your found any solution?

**Comment by kentkobi** [comment]
Hi @bloggerklik, I believe you'll also need to update the app.config.js to have appStoreUrl for ios, and playStoreUrl for android.
[StoreReview Expo documentation](https://docs.expo.dev/versions/latest/sdk/storereview/#storereviewhasaction)

**Comment by oblador** [comment]
Fixed in 0.4.1
